src/config.xml

```diff
@@ -3332,6 +3332,14 @@ to be found in the default search path.
  If you specify a
  different font using \ref cfg_dot_fontname "DOT_FONTNAME" you can set the path where \c dot
  can find it using this tag.
+]]>
+      </docs>
+    </option>
+    <option type='string' id='DOT_SHAPE' format='string' defval='plain' depends='HAVE_DOT'>
```

Comment:
Note the shape "plain" is unknown in my version of dot (2.38), "plaintext" is known.